articles/native-platforms/ios-objc/05-linking-accounts.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Linking Accounts
+description: This tutorial will show you how to link different user profiles, allowing different ways of signing in into a single profile.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objC',
+  pkgFilePath: 'iOS/basic-sample-objc/ObjCSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### Let's begin
+
+#### Be familiar with Auth0
+
+This tutorial assumes you are already familiar with Auth0 and how to Sign up and Sign in using Lock or Auth0 Toolkit. **If you're not sure, check out [this tutorial](01-login.md) first.**
+
+### 1. Setting up
+
+For this tutorial we are going to use the Auth0 Toolkit, and import the toolkit (that is writen in Swift) into our Objective-C project. All the necessary steps for this are covered on the [Custom Login](02-custom-login.md) tutorial. 
+
+### 2. Getting your second token 
+
+To link two user profiles, you need to be signed in with the user you want to use as your primary profile, the one you want to merge the other user into. Once you're signed in, you need to show the login for the second user profile, for which you have two options.
+
+#### i. Use Lock 
+
+You can use Lock, present the Auth0 sign in UI where the user can select what form of sign in he wants to use.
+
+```objc
+    A0LockViewController *controller = [lock newLockViewController];
+    controller.onAuthenticationBlock = ^(A0UserProfile *profile, A0Token *token) {
+        // dismiss the ViewController
+        [self dismissViewControllerAnimated:YES completion:nil];
+        // Link with the other user profile
+    };
+    
+    [self presentViewController:controller animated:YES completion:nil];
+
+```
+
+#### ii. Use Auth0 Toolkit
+
+You might choose to avoid the Lock UI and only show the sign in options of third parties. If that's the case, you're encouraged to read the [Custom Login](02-custom-login.md) tutorial. Anyway, here is a quick outline.
+
+- Check that you have set up the URL Type with your app's bundle ID.
+- Check that you have on your `Application Support` panel, on the Auth0 web site, the `Allowed callback URLS` set up to handle this callback:
+  {application.bundleID}://{application.domain}/ios/{application.bundleID}/callback
```

Comment:
@sebacancinos Display this as code: `{application.bundleID}://{application.domain}/ios/{application.bundleID}/callback`